### PR TITLE
Refactor load scripts

### PR DESCRIPTION
**Pull request by cuducos**
Now we use a public available CSV files, from trusted sources (official, or repositories loading from official sources using free and open-source software).

### CODE REVIEW

File: perfil/person/management/commands/load_people.py [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+        return grouped
+
+    def serialize(self, reader):
+        print('Cleaning the dataset…')
+        data = self.group_names_by_cpf(reader)
+
+        print('Importing data…')
+        for cpf, names in data.items():
+            if len(names) == 1 or probably_same_entity(names):
+                *_, name = names  # last should be the most recent one
```
Comment (leportella) [inline perfil/person/management/commands/load_people.py]: didn't knew about this `*_` thing... totally crazy!

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # api
-python-decouple
 django
-restless
 psycopg2
+python-Levenshtein
```
Comment (leportella) [inline requirements.txt]: this should be fixed, since we did it on the last 2 PRs

Comment (cuducos) [reply requirements.txt]: True story! Fixed in f9ea49a.

File: perfil/person/management/commands/load_people.py (hunk 1 above)
Comment (cuducos) [reply perfil/person/management/commands/load_people.py]: It's called packing and unpacking – pretty useful ❤️

**Pull request merged**